### DISCUSSION TITLE
Touhou Luna Nights

### DISCUSSION THREAD

**Discussion by GalladeGuy** [skyline-emu/skyline#499] [Game Requests]
I want to force Mark to play an anime game. Also it's pretty good.

**Comment by PixelyIon** [comment]
I will ensure this never happens. Sadly, @bylaws already forced me to dump "Hatsune Miku: Project DIVA MegaMix" ages ago to fix a scheduler bug and it uses a substantial amount of audren so I might be forced to interact with this anime game.

**Reply by GalladeGuy** [reply]
coward

**Comment by nekomekoraiyuu** [comment]
The game is pretty good though ngl
i always wanted to play that on phone xd
